packages/design-tokens/sass/color.scss

```diff
@@ -8,41 +8,91 @@ $kz-color-wisteria-500: #898ba9;
 $kz-color-wisteria-600: #6b6e94;
 $kz-color-wisteria-700: #4b4d68;
 $kz-color-wisteria-800: #35374a;
+$kz-color-purple-100: #f0f1f4;
```

Comment:
These are still `kz-` prefixed, should they be?

If we remove the `kz-` prefix, they then begin clashing with the names in `color-vars.scss`

Reply:
I think this deserves more consideration, because you're right, we might need to keep exposing concrete values.
@ActuallyACat lets have a discussion at some point before we make the breaking change.
@jasononeil i don't think this should be actioned in this PR because it's intended to be non-breaking.

The new `purple` `blue` etc variables are added here because i haven't excepted them from the SCSS/LESS/JSON build script. As soon as i added them to the TypeScript Theme object, they get generated.
I'm not sure it's worth adding the exception to be honest...this PR is purely a means to a smooth breaking change finale.